apps/applicant-tracking/src/components/ApplicantListItem/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { ApplicantResponse } from '@hack4impact-utk/internal-models';
+import { Chip, TableCell, TableRow } from '@mui/material';
+import LaunchIcon from '@mui/icons-material/Launch';
+import Button from '@mui/material/Button';
+
+//Each row should display the applicant name (first and last), status, NetID and application
+export default function ApplicantListItem(applicant: ApplicantResponse) {
```

Comment:
instead of making the response type the prop type do
```tsx
interface ApplicantListItemProps {
  applicant: ApplicantResponse
}

export default function ApplicantListItem({ applicant }: ApplicantListItemResponse) {
...
}
```
This lets you render the applicant list item by doing:
```tsx
<ApplicantListItem applicant={applicant} />
```

Instead of having to pass each field as a different prop.